test/acceptance/file-utils.spec.js

```diff
@@ -7,8 +7,8 @@ var os = require('os');
 var mkdirp = require('mkdirp');
 var rimraf = require('rimraf');
 
-describe('lookupFiles', function () {
-  var tmpDir = path.join(os.tmpDir(), 'mocha-lookup-files');
+describe('files utils', function () {
+  var tmpDir = path.join(os.tmpDir(), 'mocha-files');
```

Comment:
`'mocha-files'` isn't any less function-specific since the other function's name is just `.files`, it's just more vague (outside of the context of the function names anyway) and less resistant to name collision; I'd recommend leaving this directory name as `'mocha-lookup-files'` or, if you really want to avoid implying it related only to one of the two functions, changing to `'mocha-file-lookup'`.